client/my-sites/upgrades/checkout-thank-you/purchase-detail.jsx

```diff
@@ -9,47 +9,50 @@ import React from 'react';
  */
 import Button from 'components/button';
 
-const PurchaseDetail = ( {
-	additionalClass,
-	buttonText,
-	description,
-	href,
-	isPlaceholder,
-	target,
-	title
-} ) => {
-	const classes = classNames( 'checkout-thank-you__purchase-detail', additionalClass, {
-		'is-placeholder': isPlaceholder
-	} );
+const PurchaseDetail = React.createClass( {
+	propTypes: {
```

Comment:
Do you think its worth making it stateful just to have this extra function? Couldn't we encapsulate that in an `if` statement?